src/dom/root-element.ts

```diff
@@ -0,0 +1,2 @@
+export const ROOT_ELEMENT = document.children[0];
```

Comment:
What's the role of this guy?

Reply:
And also the scroll-element?  Is this just pure caching of the reference?

Reply:
Yes, purely caching. Moved SCROLL_ELEMENT to a function, ditched ROOT_ELEMENT.